.github/workflows/main.yml

```diff
@@ -469,7 +469,7 @@ jobs:
         python: [3.8] # Use flaky tests to run against more versions of Python.
         # integration: Tests with VS Code, Python extension & real Jupyter
         # notebook: Notebook Tests with VS Code, Python extension & real Jupyter
-        test-suite: [integration, notebook, notebookWithoutPythonExt, conda]
+        test-suite: [integration, notebook, remoteNotebook, notebookWithoutPythonExt, conda]
```

Comment:
Run native notebook tests with remote jupyter as a separate job (faster & easier to manage)